ui/web-v2/apps/admin/src/config/index.ts

```diff
@@ -7,6 +7,10 @@ export const urls = {
     process.env.RELEASE_CHANNEL !== 'prod'
       ? `${process.env.NX_DEV_AUTH_REDIRECT_ENDPOINT}/auth/callback`
       : `${window.location.origin}/auth/callback`,
+  GOOGLE_ANALYTICS_ID:
+    process.env.GOOGLE_ANALYTICS_ID !== 'prod'
```

Comment:
The 'prod' value comes from the environment variable `RELEASE_CHANNEL`, not from `GOOGLE_ANALYTICS_ID`.
We don't need to check `GOOGLE_ANALYTICS_ID`.

Also, the `GOOGLE_ANALYTICS_ID` is not a URL. it should be in another const.